plutus-use-cases/src/Language/Plutus/Tutorial.lhs

```diff
@@ -0,0 +1,159 @@
+plutus-use-cases: PlutusTx tutorial
+
+> {-# LANGUAGE DataKinds         #-}
+> {-# LANGUAGE RecordWildCards   #-}
+> {-# LANGUAGE TemplateHaskell   #-}
+> {-# LANGUAGE DeriveGeneric     #-}
+> {-# OPTIONS -fplugin=Language.PlutusTx.Plugin -fplugin-opt Language.PlutusTx.Plugin:dont-typecheck #-}
+> module Language.Plutus.Tutorial where
+
+This tutorial shows how to implement a simple crowdfunding campaign as a Plutus contract. We need some imports:
+
+> import           GHC.Generics (Generic)
+> import qualified Language.PlutusTx          as PlutusTx
+> import qualified Language.Plutus.Runtime    as Runtime
+> import qualified Language.Plutus.Runtime.TH as Runtime
+> import qualified Wallet.UTXO                as UTXO
+> import           Wallet.UTXO                (Validator (..))
+> import           Prelude                    hiding ((&&))
+
+The campaign has the following parameters:
+
+* Funding target
+* End date
+* Collection deadline
+* Campaign owner
+
+If the funding target is reached at the end date, then the campaign owner may collect all the funds. If it isn't reached, or if the owner does not collect the funds before the collection deadline, then the contributors are entitled to a refund.
+
+In Haskell:
+
+> data Campaign = Campaign {
+>       fundingTarget      :: Runtime.Value,
+>       endDate            :: Runtime.Height,
+>       collectionDeadline :: Runtime.Height,
+>       campaignOwner      :: Runtime.PubKey
+>  } deriving (Generic)
+
+The type of monetary values is `Runtime.Value`. Dates are expressed in terms of blocks, and they type is `Runtime.Height`. The campaign owner is identified by their public key.
```

Comment:
"they type"

Reply:
It would be good to something at this point about what these "Runtime" things are.